config/testdata/consumer_config.yml

```diff
@@ -56,14 +56,16 @@ protocol_conf:
     session_timeout: "20s"
     pool_size: 64
     pool_ttl: 600
+    gr_pool_size: 1000
+    queue_len: 128
+    queue_number: 10

```

Comment:
gr_pool_size: 1000
    queue_len: 64
    queue_number: 200

Reply:
set `gr_pool_size`=1000, `queue_number`=200 means there are only 5 goroutines to process tasks on one channel, which may cause latency issue. I think `gr_pool_size` should be 50 times greater than the `queue_number`.